resources/views/themes/elements/components/field-details.blade.php

```diff
@@ -38,7 +38,7 @@ class="svg-inline--fa fa-chevron-right fa-fw fa-sm sl-icon" role="img"
         @endif
         @if(!empty($enumValues))
             Must be one of:
-            <ul style="list-style-position: inside; list-style-type: square;">{!! implode(" ", array_map(fn($val) => "<li><code>$val</code></li>", $enumValues)) !!}</ul>
+            <ul style="list-style-position: inside; list-style-type: square;">{!! implode(" ", array_map(fn($val, $key) => "<li><code>$val ($key)</code></li>", $enumValues, array_keys($enumValues))) !!}</ul>
```

Comment:
How does this render for non-backed enums? This looks like it would result in "0 (0), 1 (1)".